product_docs/docs/biganimal/release/getting_started/01_check_resource_limits.mdx

```diff
@@ -4,35 +4,35 @@ title: Raising Azure resource limits
 
 By default, Azure sets a very low limit on the number of virtual machines and cores per region and on the number of Public IP Addresses per region that are available in a given subscription.
 
-The default number of total cores per subscription per region is 20. See [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) for more information. You need to increase the total cores to account for the total number of cores you expect to use in each region and reserve these cores to be used by EDB. See the next section [Virtual Machine Quota Requirements](#virtual-machine-quota-requirements)for more information.
+The default number of total cores per subscription per region is 20. See [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) for more information. You need to increase the total cores to account for the total number of cores you expect to use in each region and reserve these cores to be used by EDB. See [Virtual Machine Quota Requirements](#virtual-machine-quota-requirements) for more information.
 
-The default Public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your EDB clusters with the total number of EDB clusters you plan to use.
+The default Public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` with the total number of EDB clusters you plan to use for the regions where you plan to deploy your EDB clusters.
```

Comment:
The original statement was inaccurate to begin with, but i think we can simplify as follows:

"You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` to the total number of clusters you plan to deploy in a particular region."